### PR TITLE
#487 Update fixes for README.md

### PR DESCRIPTION
**Pull request by ggoebel**

### CODE REVIEW

**Comment by ThadHouse** [comment]
The instructions should be the same for any previous version. No need to special case specifically older then 4.7

Also we need to add a note that for wpilib projects, just upgrading that number is not enough to update a project, and that there are instructions of the FRC docs how to import your project into a new year.

**Comment by ThadHouse** [comment]
Actually, after thinking more, we'd rather just remove those instructions entirely, and instead point people to the wpilib docs on how to import a project into a new year.

**Comment by ggoebel** [comment]
What about mid season updates? The current GradleRIO version is 2021.2.1. 

The kickoff release import and import docs are specific to 2021.1.2.

Is the possibility of upgrading GradleRIO something that needs to be discouraged/hidden?

**Comment by ThadHouse** [comment]
I think stating that updating gradlerio with just the version number for mid season updates is fine, but between years please see the docs.

The reason we do that is because between years its possible that major incompatible changes to the build scripts are made. Especially with 2022 being a CS change, expect larger changes to happen.

**Comment by ggoebel** [comment]
updated to bring edits into line with ThadHouse's recommendations.

File: README.md [hunk 1]
```diff
@@ -108,14 +108,9 @@ Finally, you can generate and open your solution with the following command:
 Please see the gradle guide on building native software for help: https://docs.gradle.org/current/userguide/native_software.html#native_binaries:visual_studio
 
 ## Upgrading
-To upgrade your version of GradleRIO, you must first upgrade gradle. Near the bottom of your build.gradle, change the wrapper version to the following, and then run `./gradlew wrapper`:
-```gradle
-task wrapper(type: Wrapper) {
-    gradleVersion = '5.0'
-}
-```
+To upgrade your Gradle project (and GradleRio plugin) from one year to the next follow the [Importing a Gradle Project](https://docs.wpilib.org/en/stable/docs/software/vscode-overview/importing-gradle-project.html) instructions in the WPILib Documentation.
```
Comment (Starlight220) [inline README.md:111]: Perhaps a note that the import/upgrade is not in-place?

**Review by Starlight220 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sciencewhiz) [reply README.md:111]: I don't think that's necessary, as it's already on the frc-docs page.

**Review by sciencewhiz — APPROVED** [review]

**Pull request merged**